### PR TITLE
Update models

### LINKED ISSUE #18: Create Django Models

### PR DESCRIPTION
**Pull request by hiufungk**
resolves #18 

updated models and removed dummy post

### CODE REVIEW

File: backend/api/models.py [hunk 1]
```diff
@@ -1,22 +1,94 @@
...
-        return self.text
+        return self.name
 
-class AuthorProfile(models.Model):
-    author = models.ForeignKey(User, on_delete=models.CASCADE)
-    github = models.CharField(max_length=255)
 
+# model for a comment
+class Comment(models.Model):
+    author = models.ForeignKey(User, related_name="comments", on_delete=models.CASCADE)
```
Comment (forgeno) [inline backend/api/models.py]: Doesn't each comment need to have a foreign key to a post

File: backend/api/models.py [hunk 2]
```diff
@@ -1,22 +1,94 @@
...
+    def __str__(self):
+        return self.authorA + "_" + self.authorB
+
+
+# model for storing personal info of the author, linked by User model
+class AuthorProfile(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    host = models.URLField()
+    displayName = models.CharField(max_length=100)
+    github = models.URLField()
```
Comment (forgeno) [inline backend/api/models.py:91]: Missing bio and email columns

```
"github": "http://github.com/lara",
  "firstName": "Lara",
  "lastName": "Smith",
  "email": "lara@lara.com",
  "bio": "Hi, I'm Lara"
```

Comment (hiufungk) [reply backend/api/models.py:91]: added bio, email is not needed since user model provides that

File: backend/api/models.py (hunk 1 above)
Comment (hiufungk) [reply backend/api/models.py]: fixed (redid) all the flipped relationships

**Review by htruong4 — APPROVED** [review]

**Review by forgeno — APPROVED** [review]

**Pull request merged**